### PR TITLE
study view: tilt all barcharts tick labels

### PR DESCRIPTION
**Pull request by kalletlak**
# What? Why?
Fix https://github.com/cBioPortal/cbioportal/issues/6608

**Before**
Screen Shot 2019-09-16 at 1 03 35 PM

**After**
Screen Shot 2019-09-16 at 1 03 11 PM

### CODE REVIEW

File: src/pages/studyView/charts/barChart/BarChart.tsx [hunk 1]
```diff
@@ -135,12 +135,13 @@ export default class BarChart extends React.Component<IBarChartProps, {}> implem
 
     @computed
     get tilted() {
-        return this.tickValues.length > STUDY_VIEW_CONFIG.thresholds.escapeTick;
+        return _.some(this.categories,category=> category.length > 6) ||
```
[the remark this answers is not in the document]
Comment (zhx828) [reply src/pages/studyView/charts/barChart/BarChart.tsx]: Yeah, but the problem could also be caused by a large number

[the remark this answers is not in the document]
Comment (alisman) [reply src/pages/studyView/charts/barChart/BarChart.tsx]: @kalletlak  @zhx828  is there a reason not to always tilt?

[the remark this answers is not in the document]
Comment (zhx828) [reply src/pages/studyView/charts/barChart/BarChart.tsx]: @alisman we never discussed that option. I always thought the ticks showing horizontally would be easier to read. But maybe worth a discussion to keep the style consistent.

[the remark this answers is not in the document]
Comment (kalletlak) [reply src/pages/studyView/charts/barChart/BarChart.tsx]: @jjgao @zhx828 @alisman should I go ahead and always tilt labels for barplots?

**Comment by alisman** [comment]
@kalletlak ok, making a executive decision, yes, always tilt them

**Pull request merged**